docs/Transformers.md

```diff
@@ -0,0 +1,77 @@
+# Transformers
+
+A transformer allows you to access and perform transformative operations on an asset.
+
+Transformers are defined in: `./packages/configs/default/index.json`
+
+Transformers are ran in order by file type. It is recommended to have more specialized transformers first, and have them skip assets that don't match their case.
+
+## Transformer Life Cycle
+
+Below is the list of the life cycle methods in the order they are called.
+
+### `getConfig`
+
+Loads configuration specifically for this transformer.
+
+**API**
+
+```
+getConfig ({
```

Comment:
In the long run we should probably be extracting API docs like this from the source